### PR TITLE
test : check if gopacket didn't panic

### PR DESCRIPTION
**Pull request by nplanel**

### CODE REVIEW

File: flow/flow_test.go [hunk 1]
```diff
@@ -321,6 +321,17 @@ func compareFlow(expected, tested *Flow) bool {
 	return true
 }
 
+func checkGoPacketSanity(t *testing.T, p gopacket.Packet) {
+	if errLayer := p.ErrorLayer(); errLayer != nil {
+		if l := p.Layer(gopacket.LayerTypeDecodeFailure); l != nil {
+			df := l.(*gopacket.DecodeFailure)
+			if len(df.Dump()) > 0 {
```
Comment (safchain) [inline flow/flow_test.go]: if df.Dump() doesn't return anything we won't get the error.

Comment (nplanel) [reply flow/flow_test.go]: yes that the purpose, as DecodeFailure layer can be used to return an "legit" error like : "packet too small" but didn't crash/panic
df.Dump() return something only if a panic occured during a protocol DecodingLayer()

**Comment by nplanel** [comment]
run  skydive-scale-tests

**Comment by nplanel** [comment]
run  skydive-scale-tests

**Comment by nplanel** [comment]
run scale-tests

File: flow/flow_test.go (hunk 1 above)
Comment (safchain) [reply flow/flow_test.go]: ok thanks for the explanation

**Pull request merged**